simulator/tests/action_tests.rs

```diff
@@ -90,7 +91,7 @@ fn test_trained_eye_opener() {
     let state = SimulationState::from_macro(&SETTINGS, &[Action::TrainedEye]);
     assert!(matches!(state, Ok(_)));
     let state = state.unwrap();
-    assert_eq!(state.missing_quality, 0);
+    assert_eq!(state.get_missing_quality(), 0);
```

Comment:
We should test that using `TrainedEye` under Poor condition still gives max quality.